pages/sign-in/kakao/index.page.tsx

```diff
@@ -7,33 +7,34 @@ import { instance } from "@/apis/axios";
 import { PAGE_PATH } from "@/constants/pagePath";
 
 interface KakaoProps {
-  isFirst: boolean;
+  id: number;
+  isComplete: boolean;
 }
 
 export async function getServerSideProps(
   context: GetServerSidePropsContext,
 ): Promise<GetServerSidePropsResult<KakaoProps>> {
-  const code = context.query["code"];
+  const { code } = context.query;
 
   const res = await instance.get(`/login/oauth/kakao?code=${code}`);
-  const { isFirst } = res.data;
+  const { id, isComplete } = res.data;
 
   return {
-    props: { isFirst: Boolean(isFirst) },
+    props: { id, isComplete },
   };
 }
 
-export default function Kakao({ isFirst }: KakaoProps) {
+export default function Kakao({ id, isComplete }: KakaoProps) {
   const router = useRouter();
 
   useEffect(() => {
-    if (isFirst) {
-      router.push(PAGE_PATH.PROFILE_SETUP);
-      return; // TODO: 우혁님 PR merge 된 후에, 상수로 수정 예정
+    if (isComplete === false) {
+      router.push({ pathname: PAGE_PATH.PROFILE_SETUP, query: { id } }, PAGE_PATH.PROFILE_SETUP);
+      return;
     }
 
     router.push(PAGE_PATH.MAIN);
```

Comment:
```suggestion
    router.push(PAGE_PATH.DASHBOARD);
```
p3: 유저가 추가 정보 입력한 적이 있으면 대시보드 페이지로 이동하면 될 것 같아요